source/helper.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  FileDetailProperty,
+  AsyncFunction,
+} from './types';
+
+import {
+  // Exception as NextcloudError,
+
+  ForbiddenError,
+  NotFoundError,
+} from './errors';
+
+const sanitizePath = encodeURI;
+
+export function createFileDetailProperty(namespace: string, namespaceShort: string, element: string, nativeType?: boolean, defaultValue?: any) : FileDetailProperty {
```

Comment:
What do our code style guidelines say about a large number of parameters and breaking them onto individual lines?